README.md

```diff
@@ -1,112 +1,113 @@
 # PromptSource
-Promptsource is a toolkit for collecting and applying prompts to NLP datasets.
+**PromptSource is a toolkit for creating, sharing and using natural language prompts.**
 
-Promptsource uses a simple templating language to programatically map an example of a dataset into a text input and a text target.
+Prompts are functions that map an example from a dataset to a natural language input and target output PromptSource contains a growing collection of prompts (which we call **P3**: **P**ublic **P**ool of **P**rompts). As of January 20, 2022, there are ~2'000 English prompts for 170+ English datasets in [P3](https://huggingface.co/datasets/bigscience/P3).
 
-Promptsource contains a growing collection of prompts (which we call **P3**: **P**ublic **P**ool of **P**rompts). As of October 18th, there are ~2'000 prompts for 170+ datasets in [P3](https://huggingface.co/datasets/bigscience/P3).
-Feel free to use these prompts as they are (you'll find citation details [here](#Citation)).
-
-Note that a subset of the prompts are still *Work in Progress*. You'll find the list of the prompts which will potentially be modified in the near future [here](WIP.md). Modifications will in majority consist of metadata collection, but in some cases, will impact the templates themselves. To facilitate traceability, Promptsource is currently pinned at version `0.1.0`.
-
-Propmtsource and P3 were originally developed as part of the paper [Multitask Prompted Training Enables Zero-Shot Task Generalization](https://arxiv.org/abs/2110.08207). We release T0* (pronounce "T Zero"), a series of model trained on [P3](https://huggingface.co/datasets/bigscience/P3). Checkpoints are available [here](https://huggingface.co/bigscience/T0pp). In particular, we recommend using T0++ (pronounce "T Zero Plus Plus") as it leads (on average) to the best performances on a variety of NLP tasks.
-
-**You will find the official repository to reproduce the results of T Zero here: https://github.com/bigscience-workshop/t-zero.**
+PromptSource provides the tools to create, and share natural language prompts (see [How to use prompts](#how-to-use-prompts), and then use the thousands of existing and newly created prompts through a simple API (see How to use prompts).
+Prompts are saved in standalone structured files and are written in a simple templating language called Jinja.
 
```

Comment:
Needs an example of a prompt here. Also we should include figure 1 from the paper, it is awesome way to explain what is going on.